crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -273,26 +271,34 @@ def nodes
           remaining = NodeObject.find(
             "state:crowbar_upgrade AND NOT run_list_map:ceph_*"
           ).size
-          ::Crowbar::UpgradeStatus.new.save_nodes(0, remaining)
+          status.save_nodes(0, remaining)
```

Comment:
Hm, I'm not sure if this is gonna work. I think I was using the original object in my first tests, but then went to creating new one because the data were rewritten from some other source (I think from `api/nodes`)